deployment/hasura/metadata/actions.graphql

```diff
@@ -96,7 +96,8 @@ type Query {
 
 type Query {
   constraintsDslTypescript(
-    missionModelId: ID!
+    missionModelId: ID!,
+    planId: Int
```

Comment:
Can this `planId` possibly be null? Or should this be `Int!`?

Reply:
Yes, it is supposed to optional. If provided, plan-specific code is add to the mission-specific code.